server/store_statistics.go

```diff
@@ -101,6 +101,14 @@ func (s *storeStatistics) Observe(store *core.StoreInfo) {
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_available").Set(float64(store.GetAvailable()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_used").Set(float64(store.GetUsedSize()))
 	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_capacity").Set(float64(store.GetCapacity()))
+
+	// Store flows.
+	storeBytesWriteRate, storeBytesReadRate := store.GetRollingStoreStats().GetBytesRate()
+	storeStatusGauge.WithLabelValues(s.namespace, storeAddress, id, "store_bytes_write_rate").Set(float64(storeBytesWriteRate))
```

Comment:
According to the best practice of Prometheus's name ruling, is `store_write_rate_bytes`  preferred than `store_bytes_write_rate`?

Refer to: https://prometheus.io/docs/practices/naming/#metric-names

Reply:
I think the name is the display name, not variable name.